python/python/glide/async_commands/redis_modules/search/field.py

```diff
@@ -0,0 +1,375 @@
+from enum import Enum
+from typing import Dict, List, Optional
+
+
+class SortableOption(Enum):
+    """
+    Sortable options: options for fields with SORTABLE argument.
+    - SORTABLE - Sets the field as sortable.
```

Comment:
Change throughout this PR, in all new enums: this is not how enums are being documented in python. Please see config.py::ReadFrom for example